pallas-primitives/src/conway/model.rs

```diff
@@ -1220,7 +1220,23 @@ where
     }
 }
 
-pub use crate::babbage::PseudoPostAlonzoTransactionOutput;
+#[derive(Encode, Decode, Debug, PartialEq, Eq, Clone)]
+#[cbor(map)]
+pub struct PseudoPostAlonzoTransactionOutput<T1, T2> {
+    #[n(0)]
+    pub address: Bytes,
+
+    #[n(1)]
+    pub value: Value,
+
+    #[n(2)]
+    pub datum_option: Option<T1>,
+
+    #[n(3)]
+    pub script_ref: Option<CborWrap<T2>>,
+}
+
```

Comment:
what's the difference with the babbage one?

Reply:
Ah no! The `Value` it is!

Reply:
We could also just parameterize over the Value.